### PR TITLE
Revert "Bump go to 1.14 (#2853)"

### PR DESCRIPTION
**Pull request by vdarulis**
This reverts commit 55e6e44bd28ee9f80c8f6120147c15daa82193f8.

**What this PR does / why we need it**:

Go back to 1.13 while regressions are being investigated.

**Special notes for your reviewer**:

**Does this PR introduce a user-facing and/or backwards incompatible change?**:

```release-note

```

**Does this PR require updating code package or user-facing documentation?**:

```documentation-note

```

### CODE REVIEW

**Review by mway — APPROVED** [review]

**Review by robskillington — APPROVED** [review]
LGTM, look forward to after getting sign off from multiple partners that 1.14 looks good to restoring this (as discussed we may very well go straight to testing 1.15 instead post-1.0)

**Pull request merged**
